app/src/main/java/com/android/unio/ui/association/AssociationProfile.kt

```diff
@@ -137,14 +131,22 @@ fun AssociationProfileScaffold(
       },
       topBar = {
         TopAppBar(
-            title = { Text(title, modifier = Modifier.testTag("AssociationProfileTitle")) },
+            title = {
+              val title: String
+              if (association != null) {
+                title = association.fullName
```

Comment:
As I said in above, I think that it would look better and more coherent if we used association.name here instead.